src/themes/hugo-theme-learn/static/js/hookFilter.js

```diff
@@ -0,0 +1,53 @@
+$(function() {
+  const allItems = $('#hookFilter ~ dl dt');
+
+  const escapeRegExp = function(string) {
+    return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+  };
+
+  const displayAll = function() {
+    allItems.each(function() {
+      const $item = $(this);
+      if ($item.is(':hidden')) {
+        $item.show();
+        $item.next('dd').show();
+      }
+    });
+  };
+
+  const filterList = function(search) {
+    let count = 0;
+    const regex = new RegExp(escapeRegExp(search), 'i');
+
+    allItems.each(function() {
+      const $item = $(this);
+      const $sibling = $item.next('dd');
+      const text = $item.text();
+      const matchesSearch = Boolean(text.match(regex));
+      $item.toggle(matchesSearch);
+      $sibling.toggle(matchesSearch);
+
+      if (matchesSearch) {
+        count += 1;
+      }
+    });
+
+    if (count === 0) {
+      $('#hookFilter .empty').addClass('show');
+    } else {
+      $('#hookFilter .empty').removeClass('show');
+    }
```

Comment:
```suggestion
    $('#hookFilter .empty').toggleClass('show', count === 0);
```

Reply:
In his condition the class `show` is only added if `count === 0`, with `toggleclass` it could remove the class instead, if it was present before, so the behavior is not exactly the same. Is this really the behavior we want ?